app/helpers/application_helper.rb

```diff
@@ -40,15 +40,20 @@ def policies_areas_data_container
     Policy.areas.map { |policy| [policy.name, policy.id] }
   end
 
-  # Re-orders the data container such that +selected+ ones appear first.
+  # Re-orders the data container such that +selected+ ones appear first.
+  # and both in alphabetical order
   def prioritise_data_container(unprioritised_container, selected)
```

Comment:
I can't think of a better name for this, certainly it didn't make much sense to me until I realised what it was doing, but this method receives a different data, `organisation_data_container', `people_data_container`, `working_groups_data_container`, `policies_areas_data_container` which are basically Arrays of Arrays with `title` and `content_id`.

I didn't wanted to move it around as this app seems to be using only ApplicationHelper and I would like to not cause more noise in this PR.